server/resources/translations/common.en.json

```diff
@@ -14,5 +14,11 @@
   "sensorIsTampered": "The door sensor at {{ deviceDisplayName }} is not fully attached to the door. If you require a replacement door sensor or have any questions about this, contact us by emailing clientsupport@brave.coop.",
   "sensorLowBatteryInitial": "The battery for the {{ deviceDisplayName }} door sensor is low, and needs replacing.\n\nTo watch a video showing how to replace the battery, go to https://youtu.be/-mfk4-qQc4w",
   "sensorReconnection": "The Brave Sensor at {{ deviceDisplayName }} ({{ clientDisplayName }}) has been reconnected.",
-  "thankYou": "Thank you"
+  "thankYou": "Thank you",
+  "SENSOR_DURATION": "Duration",
+  "SENSOR_STILLNESS": "Stillness",
+  "SENSOR_UNKNOWN": "Unknown",
+  "Cat0": "Cat0",
+  "Cat1": "Cat1"
```

Comment:
`Cat0` and `Cat1` are only used for tests. They should not be included in the files used for production. Can you please take these out? If your tests then begin you fail, you may need to mock out the `t` function or use real categories.